src/pages/hegel/reference/mechanical-object-section-1.mdx

```diff
@@ -0,0 +1,18 @@
+
+In a somewhat throwaway remark at the end of this conceptual discussion, Hegel writes that the “mechanical object” “does not consist of *atoms*, for these are not objects because they are not totalities” (712).  It is worth musing on this brief remark because it reveals a lot about how Hegel conceptualises the wider significance of his account of the “mechanical object”. Hegel clearly has some mechanistic, in particular atomistic, worldviews in his crosshairs when he goes out of his way to state that the “mechanical object” does not consist of atoms. Hegel’s target is those that theorise that the world is fundamentally made out of atoms and that since the world is made out of atoms that everything is really just atoms.  In Hegel’s account of “Mechanism”, he goes out of his way to explicitly state that the “mechanical object”, the simplest kind of mechanical entity, is not reducible to atoms. There is not something more fundamental to the “mechanical object” that is the true reality of things. Moreover, and perhaps a stronger rebuke to the atomists, not only is the “mechanical object” not reducible to atoms, but it is also not the most fundamental thing. Hegel rejects this notion of fundamentality out of hand and it is this rejection that is his strongest argument against atomistic or atomistic-like conceptions of reality.
```

Comment:
1. "theorise that the world is fundamentally made out of atoms and that since the world is made out of atoms that everything is really just atoms" => sure, but is the `world` really at question here? The real target is the nature of mechanical object. I suppose that you could say if the mechanical object as such cannot be understood as mere atoms, the neither can the world if it is only made up of mechanical objects. 
2. About the issue of "fundamentality". Even if one grants that reality isn't fundamentally made up of atoms, one could still argue that the mechanical object is fundamentally made up of atoms (and thereby assume that something else more fundamental is the further "bottom of reality"). I think Hegel wants to say that no matter the fundamental aspect of reality/world/etc, the mechanical object is not reducible to atoms. Regardless, can we mount a better support of Hegel's claim here, if at all possible?

Reply:
1. Well, insofar as atomistic theories are theories about reality, I think that Hegel is definitely saying something about reality or the world when he is remarking that the mechanical object is not made up of atoms. Moreover, isn't he saying something about the world all the time in the SL? If it is a study of the fundamental determinations of Being and Thought, then our understanding of each of these determinations goes someway towards our understand of the world or reality.
2. Well, first of all, I think that Hegel's argument against atomism can be more generally captured by his argument against fundamentality, which is what I try to explain in the final sentence. I suppose I could say more about this, but my concern would be that it might take us too far away from Mechanism. Could be hyperlinked to a Guide article about fundamentality? But it is this more general stance against fundamentality that is Hegel's proper attack on a theory like atomism. 
I think he mentions it here because we are in Mechanism and atomism is a mechanistic worldview. I think you're right to say that Mechanism itself is not a rebuke to atomism. It is a rebuke in the sense that Hegel's account of Mechanism does not permit an atomistic theory of the world, but it is not a direct attack on atomism. That attack must be found in his critique of foundationalism.
So to summarise, I can edit the final sentences to better express the point. Thanks!

Reply:
1. Saying that SL says something about the world is a little imprecise given that the `world` should have some sort of categorical definition, but it doesn't. I think it's fine to refer to the world loosely to help our understanding, but to use it as a technical leverage is thin since the concept is pretty vague. In one sense, mechanism does address this vagueness since it aims to determine a grand totality coherently unto itself. 
2. Ok, thanks ✅